clickhouse-admin/types/src/lib.rs

```diff
@@ -16,8 +21,20 @@ use std::io::{ErrorKind, Write};
 use std::net::Ipv6Addr;
 use std::str::FromStr;
 
-pub mod config;
-use config::*;
+mod config;
+pub use config::{
+    ClickhouseHost, KeeperConfig, KeeperConfigsForReplica, KeeperNodeConfig,
+    LogConfig, LogLevel, Macros, NodeType, RaftServerConfig,
+    RaftServerSettings, RaftServers, ReplicaConfig, ServerNodeConfig,
+};
+
+// Used for schemars to be able to be used with camino:
+// See https://github.com/camino-rs/camino/issues/91#issuecomment-2027908513
+pub fn path_schema(gen: &mut SchemaGenerator) -> Schema {
```

Comment:
😭

Reply:
LOL. It's everyone's* favorite issue reference!

*Rain's favorite